addons/binding/org.openhab.binding.harmonyhub/src/main/java/org/openhab/binding/harmonyhub/internal/handler/HarmonyHubHandler.java

```diff
@@ -85,19 +81,23 @@
     private static final long CONFIG_CACHE_TIME = TimeUnit.MINUTES.toMillis(1);
     private static final int RETRY_TIME = 60;
     private static final int HEARTBEAT_INTERVAL = 30;
+    // Websocket will timeout after 60 seconds, pick a sensible max under this,
+    private static final int HEARTBEAT_INTERVAL_MAX = 50;
 
     private List<HubStatusListener> listeners = new CopyOnWriteArrayList<>();
 
     private final ExpiringCacheAsync<@Nullable HarmonyConfig> configCache = new ExpiringCacheAsync<>(CONFIG_CACHE_TIME);
     private final HarmonyHubHandlerFactory factory;
 
-    private @NonNullByDefault({}) ScheduledExecutorService buttonExecutor;
+    // private @NonNullByDefault({}) ScheduledExecutorService buttonExecutor;
```

Comment:
If it's no longer used the commented code should be removed.

Reply:
👍